src/blob.c

```diff
@@ -70,11 +70,12 @@ Blob_diff(Blob *self, PyObject *args, PyObject *kwds)
     char *old_as_path = NULL, *new_as_path = NULL;
     Blob *other = NULL;
     int err;
-    char *keywords[] = {"blob", "flag", "old_as_path", "new_as_path", NULL};
+    char *keywords[] = {"blob", "flag", "old_as_path", "new_as_path", "context_lines", "interhunk_lines", NULL};
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|O!Iss", keywords,
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|O!IssHH", keywords,
                                      &BlobType, &other, &opts.flags,
-                                     &old_as_path, &new_as_path))
+                                     &old_as_path, &new_as_path,
+                                     &opts.context_lines, &opts.interhunk_lines))
```

Comment:
This adds 2 new keyword arguments to Blob.diff but the documentation string is not updated, and please add unit tests for the new arguments.

Reply:
Just pushed an amended commit with the required adjustments.